app/scripts/add-test-accounts-for-integration-tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -ex
+here=$(dirname $0)
+
+$here/cli.sh add "Test User" test.user \
+    test@example.com password \
+    --if-not-exists
```

Comment:
Idempotent, so multiple batches of portal integration tests can be run against the same database.